src/main/java/spleetwaise/address/logic/commands/EditCommand.java

```diff
@@ -113,9 +115,30 @@ public CommandResult execute() throws CommandException {
 
         model.setPerson(personToEdit, editedPerson);
         model.updateFilteredPersonList(AddressBookModel.PREDICATE_SHOW_ALL_PERSONS);
+
+        updatePersonsInTransactions(model, personToEdit, editedPerson);
+
         return new CommandResult(String.format(MESSAGE_EDIT_PERSON_SUCCESS, Messages.format(editedPerson)));
     }
 
+    private void updatePersonsInTransactions(
```

Comment:
Understand that this is Draft still. Might need to add rationale for PR reviewers to understand the need to have this method

Reply:
This method is for bugfix #202 when a user edits a person name, the matching person name/s in the transaction list should get updated as well.